ArduCopter/ReleaseNotes.txt

```diff
@@ -739,7 +739,7 @@ Changes from 4.1.5
 1) AHRS/EKF improvements
     a) EKF startup messages reduced
     b) GPS<->Opticalflow automatic switching improved (see ahrs-source-gps-optflow.lua script)
-    c) LORD Microstrain CX5/GX5 external AHRS support
```

Comment:
I did leave the LORD name in the release notes, just because it could still be associated with them.

Reply:
I don't think we should update old release notes like this

Reply:
Ok, I'll revert.

Reply:
Done, those commits are dropped.